tools/go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/newrelic/terraform-provider-newrelic/v2/tools
 
-go 1.19
+go 1.21
 
 require (
 	github.com/bflad/tfproviderlint v0.29.0
 	github.com/client9/misspell v0.3.4
 	github.com/git-chglog/git-chglog v0.15.4
 	github.com/golangci/golangci-lint v1.51.2
-	github.com/goreleaser/goreleaser v1.7.0
+	github.com/goreleaser/goreleaser v1.23.0
```

Comment:
#### 2 

It may be observed that `goreleaser` has been bumped up without any compilation issues, which means this can now work with `go1.21`.